### ISSUE TITLE
Laravel Login/Logout Event Not Dispatched

### ISSUE THREAD

**Issue by nickescobedo** [auth0/laravel-auth0#328]
### SDK Version

7.0

### PHP Version

PHP 8.1

### Composer Version

2.x

### What happened?

Upon login and logout from Laravel the default Laravel login/logout events should be dispatched.

### How can we reproduce this issue?

Create a listener for login and logout. Add them in the EventServiceProvider.

Laravel has the available auth events here: https://laravel.com/docs/9.x/authentication#events

Examples
```
    'Illuminate\Auth\Events\Login' => [
        'App\Listeners\LogSuccessfulLogin',
    ],
```

```
    'Illuminate\Auth\Events\Logout' => [
        'App\Listeners\LogSuccessfulLogout',
    ],
```

The listeners aren't triggered.

### Additional context

Being able to tap into these events is crucial for firing off other items applications need to run.

**Comment by evansims** [comment]
Hey @nickescobedo 👋 Thanks for your enhancement suggestion. At present, we raise a custom event (`\Auth0\Laravel\Event\Stateful\AuthenticationSucceeded`) for successful logins, but do not have an event for log-outs. I agree that supporting these additional events make sense. I'll see what I can do on getting those integrated. 👍

**Comment by nickescobedo** [comment]
Hi @evansims Thank you for the quick reply.

Looking forward to it!

**Comment by evansims** [comment]
Hey @nickescobedo 👋 Thanks again for bringing this request up. I've added support for [a number of new events](https://github.com/auth0/laravel-auth0/pull/331), and released these changes as [7.3.0](https://github.com/auth0/laravel-auth0/releases/tag/7.3.0). Please let me know if you have any other suggestions!
